Formula/orientdb.rb

```diff
@@ -39,4 +39,14 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    begin
+      system "#{bin}/orientdb", "start"
```

Comment:
Seems like updating the `orientdb-server-log.properties` file to use `#{var}/log` would be a good idea and may fix this test also.